### PR TITLE
docker-compose 1.2.0

### PR DESCRIPTION
**Pull request by inka**
upstream upgrade

### CODE REVIEW

**Comment by saada** [comment]
:+1:

File: Library/Formula/fig.rb [hunk 1]
```diff
@@ -19,43 +19,48 @@ class Fig < Formula
...
   resource "docopt" do
     url "https://pypi.python.org/packages/source/d/docopt/docopt-0.6.2.tar.gz"
-    sha1 "224a3ec08b56445a1bd1583aad06b00692671e04"
+    sha256 "49b3a825280bd66b3aa83585ef59c4a8c82f2c8a522dbe754a8bc8d08c85c491"
   end
 
   resource "requests" do
-    url "https://pypi.python.org/packages/source/r/requests/requests-2.4.3.tar.gz"
-    sha1 "411f1bfa44556f7dd0f34cd822047c31baa7d741"
+    url "https://pypi.python.org/packages/source/r/requests/requests-2.5.3.tar.gz"
```
Comment (tdsmith) [inline Library/Formula/fig.rb]: Can this go to 2.6.0 or is something holding it back?

Comment (inka) [inline Library/Formula/fig.rb]: I tried and tested that but the upstream maintainer has set a version constraint <2.6.0.

Comment (tdsmith) [inline Library/Formula/fig.rb]: Which package?

Comment (tdsmith) [inline Library/Formula/fig.rb]: Oops, this one, you're right. Thanks!

**Comment by tdsmith** [comment]
Thanks for the pull request! 🎉 Homebrew depends on contributions from community members like you and we're grateful for your support.

**Comment by garthk** [comment]
@Inka, mind repeating the performance for `docker-compose` 1.3.0?

**Pull request closed without merging**